install.sh

```diff
@@ -133,22 +114,11 @@ adjust_os() {
   esac
   true
 }
-s3_releases() {
-  s3url="${S3_CONTENT_CHECK_URL}${PROJECT_NAME}/archives/&delimiter=/"
-  xml=$(http_copy "$s3url")
-  versions=$(echo "$xml" | sed -n 's/</\
-</gp' | sed -n "s/<Prefix>${PROJECT_NAME}\/archives\/\(.*\)\//\1/p") || return 1
-  test -z "$versions" && return 1
-  echo "$versions" | sort --version-sort
-}
-s3_release() {
-  version=$1
-  test -z "$version" && version="latest"
-  s3url="${S3_CONTENT_CHECK_URL}${PROJECT_NAME}/archives/${version#v}/&delimiter=/"
-  xml=$(http_copy "$s3url")
-  exists=$(echo "$xml" | grep "<Key>") || return 1
-  test -z "$version" && return 1
-  echo "$version"
+releases() {
```

Comment:
I ran `./install.sh -l` because I figured it's easier to parse the regex when I know what the match looks like, but I see an empty list. Running `curl -s https://api.github.com/repos/confluentinc/cli/releases` on it's own I see a rate limit error message.

Reply:
How many times did you run this? I must've ran it over 20 times, and I never got a rate limit error.

Reply:
Around 5 times.

Reply:
I just waited an hour and it works.